### ISSUE TITLE
Review packages for the whole platform and components

### ISSUE THREAD

**Issue by amedranogil** [universAAL/platform#42]
ensure the java packages follow a convention and that each component has it's packages defined according to that convention.
Ontologies already have this, but other EG surely do not.

**Comment by cstockloew** [comment]
agreed, but what convention do you mean?

A good point to start is the javadoc:
https://universaal.github.io/platform/apidocs/index.html

It contains a list of all packages. Most of it seems to be OK. A first look reveals:
- ucc should have package org.universAAL.tools.ucc.* instead of org.universAAL.ucc.*
- zigbee uses org.universAAL.hw.* instead of org.universAAL.lddi.*

Do you mean this?

**Comment by amedranogil** [comment]
yes, but with more carefull examination:

- [ ] ..commerce.ustore.tools move to ..tools
- [ ] ..configurable.module should be ..middlware.module.configuration (or similar)
- [ ] drools is inside context, reasoner
- [ ] ..lddi.adapter (?)
- [ ] ..managers,api vs modules (use single concention, I like the second, in the api docs list the api is at top)
- [ ] ..mw.tracker (?)
- [ ] all ..support may better be renamed in to better package names.
- [x] ..ui.handler.web (!) this one is easy.
- [x] ..kinect is a ui.handler
- [x] all junit packages should be something like ..middleware.container.junit{.bus|.mwtest} (actually this could also be a proposal to rename their respective project names and folders inside mw.core)
- [x] ..ioc.dependencies is an utility library
- [x] ..log, which module defines this package (WHY!??)

**Comment by cstockloew** [comment]
> all junit packages should be something like ..middleware.container.junit{.bus|.mwtest} (actually this could also be a proposal to rename their respective project names and folders inside mw.core)

I prefer to have the container (core, osgi, junit) at the end

**Comment by cstockloew** [comment]
> ..log, which module defines this package (WHY!??)

ri.gateway.multitenant, can you change it?

**Comment by amedranogil** [comment]
Yes, I remember I had this discussion with @kismet the communication layer of the MTGW is all done using standard loging, that package holds the helper methods to do that; there is quite some loging; but I'll look into it.
